test/screenshot/lib/controller.js

```diff
@@ -84,6 +91,7 @@ class Controller {
 
   async initialize() {
     this.baseUploadDir_ = await this.storage_.generateUniqueUploadDir();
+    await this.gitRepo_.fetch();
```

Comment:
Are there any implications of this being run here rather than `parseDiffBase` in `cli-arg-parser` now?

Reply:
AFAIK the only differences are:

1. It only runs once, instead of every time `parseDiffBase()` is called
    (which is why I moved it here)
2. If fetching fails for some reason, it'll happen immediately instead of after a minute of uploading files to GCS (which is probably better devex)